bundles/org.openhab.binding.hue/src/main/java/org/openhab/binding/hue/internal/handler/HueGroupHandler.java

```diff
@@ -123,6 +127,15 @@ private void initializeThing(@Nullable ThingStatus bridgeStatus) {
         }
     }
 
+    private synchronized void initializeProperties(@Nullable FullGroup fullGroup) {
+        if (!propertiesInitializedSuccessfully && fullGroup != null) {
```

Comment:
The members of a group can be changed (form the Hue app). In this case, the property will not be updated.
What is exactly the purpose of this new property in openHAB ?

Reply:
You are right. We do not need this flag. The framework checks for changes.

The property itself shows which members are part of the group.